src/components/Guilds/common/Form/DurationInput/index.tsx

```diff
@@ -0,0 +1,98 @@
+import styled from 'styled-components';
+import { GoTriangleUp, GoTriangleDown } from 'react-icons/go';
+import React from 'react';
+import { DURATION_LIMITS } from 'constants/Duration';
+import { useDuration } from 'hooks/Guilds/useDuration';
+import WarningInput from './WarningInput';
+import { Flex } from 'Components/Primitives/Layout';
+import { Button } from 'old-components/common/Button';
+import { Modal } from 'old-components/Guilds/common/Modal';
+import NumericalInput from 'old-components/Guilds/common/Form/NumericalInput';
+
+const Column = styled(Flex)`
+  flex-direction: column;
+  width: 100px;
+  margin: 20px 10px;
+`;
+
+const Container = styled(Flex)`
+  flex-direction: row;
+  justify-content: centre;
+`;
+
+const ColumnButton = styled(Button).attrs({
+  variant: 'primary',
+})`
+  background-color: transparent;
+  margin: 0;
+  border: none;
+  :hover:enabled {
+    border-color: none;
+  }
+`;
+
+const WarningRow = styled(Flex)``;

```

Comment:
If there is no customization you can delete this and go with just the Flex component.